src/styles/blocks/card.scss

```diff
@@ -0,0 +1,168 @@
+@import '../utils/variables';
+
+.cards-container {
+  box-sizing: border-box;
+
+  --column-count: 1;
+
+  padding: 40px 50px;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 46px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards-container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards-container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards-container {
+    --column-count: 4;
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  margin: 16px;
+  height: auto;
+  transition:
+    transform 300ms,
+    color 300ms;
+
+  &__image {
+    display: flex;
+
+    .image {
+      width: 160px;
+      height: 134px;
+      margin: 32px 19px 0;
+
+      background-image: url(../images/imac.jpeg);
```

Comment:
The background image is hardcoded. Consider using a variable or a mixin if this image might change or if you have multiple images to manage.